cypress/support/fragments/eholdings/eHoldingsTitle.js

```diff
@@ -16,27 +18,29 @@ export default {
   filterStatuses,
 
   filterPackages: (selectionStatus = filterStatuses.notSelected, packageName) => {
+    //Test is not working stabil if no waits.
```

Comment:
Here we need a comment why you use cy.wait here and why you can not use element expectation with interactor

Reply:
Fixed. Test is not running stable with element expectation, there is a method before this one to wait until element  is found and it is finding and clicking but the modal is not loading sometimes. When I use wait 2 seconds it is working properly.